lib/registration.pm

```diff
@@ -1075,7 +1075,10 @@ sub runtime_registration {
     if (is_transactional) {
         trup_call('register' . $cmd);
         trup_call('--continue run zypper --gpg-auto-import-keys refresh') if is_staging;
-        add_suseconnect_product('SL-Micro-Extras') if (is_sle_micro('>=6.0'));
+        if (is_sle_micro('>=6.0')) {
+            process_reboot(trigger => 1);
+            add_suseconnect_product('SL-Micro-Extras');
+        }
```

Comment:
Why make this change ?

Reply:
This code snippet is from another PR which mentioned in related ticket, it's just for testing use.

Reply:
@nanzhg The  commit is already given in another PR, please exclude this change and only include yours before merge.

Reply:
Sure, will remove it in final commit. For now, I'd like to decide how to resolve the package install conflicts.